measure_extinction/extdata.py

```diff
@@ -173,8 +175,13 @@ def AverageExtData(extdatas):
             aveext.stds[src] = np.nanstd(exts, axis=0, ddof=1)
             aveext.uncs[src] = aveext.stds[src] / np.sqrt(aveext.npts[src])
 
-        # take out the data points where less than 3 values were averaged
-        aveext.npts[src][aveext.npts[src] < 3] = 0
+        # take out the data points where less than a certain number of values was averaged, and give a warning
+        if min_number > 1:
+            aveext.npts[src][aveext.npts[src] < min_number] = 0
+            warnings.warn(
```

Comment:
If you could include the min_number value in the warning, then this would make the warning more informative.

Reply:
Done.